projects/fsastorefrontlib/src/cms-components/sync-pilot/cms-sync-pilot/cms-sync-pilot.component.ts

```diff
@@ -108,7 +110,8 @@ export class CmsSyncPilotComponent implements OnInit, OnDestroy {
         .pipe(
           withLatestFrom((guestEndpoint: GuestEndpoint) => {
             if (guestEndpoint.state === 'accepted') {
-              const url = guestEndpoint.targetChannelAddress;
+              let url = guestEndpoint.targetChannelAddress;
```

Comment:
Can this work:
```
(guestEndpoint.targetChannelAddress).replace(this.GENDER_URL_PARAM, '');
```
Instead of reassigning to the same variable?

Reply:
Yes, it'll work. I'll change that.